src/components/work-here.js

```diff
@@ -0,0 +1,25 @@
+import SectionHeader from "./section-header";
+import SectionInfo from "./section-info";
+import colors from "../styles";
+
+const WorkHere = () => {
+	const pStyle = {
+		padding: 12,
+		paddingLeft: 24,
+		lineHeight: "24px",
+		fontSize: 18,
+		color: colors.veryDarkGray
+	};
+	return (
+		<div>
+			<SectionHeader>Work here</SectionHeader>
+			<SectionInfo style={pStyle}>
+				We're always looking to bring on new people who are as excited about technology as we are.
```

Comment:
nice, i still blame github for only supporting jsx with `.jsx` extension though , https://github.com/github/linguist/blob/10e0fa4360a6a059dab76a0da661baa80b7f36dd/lib/linguist/languages.yml#L1614